python-package/lightgbm/plotting.py

```diff
@@ -536,6 +551,14 @@ def plot_tree(booster, ax=None, tree_index=0, figsize=None, dpi=None,
               show_info=None, precision=3, orientation='horizontal', **kwargs):
     """Plot specified tree.
 
+    Non-leaf nodes have labels like ``Column_10 <= 875.9``, which means
```

Comment:
To unify descriptions for `create_tree_digraph` and `plot_tree`.
```suggestion
    Each node in the graph represents a node in the tree.

    Non-leaf nodes have labels like ``Column_10 <= 875.9``, which means
```